lib/stronger_parameters/parameters.rb

```diff
@@ -194,7 +195,22 @@ module ControllerSupport
       end
     end
   end
+
+  module APIControllerSupport
+    extend ActiveSupport::Concern
+
+    Parameters = ActionController::Parameters
+
+    included do
+      # TODO: this is not consistent with the behavior of raising ActionController::UnpermittedParameters
+      # should have the same render vs raise behavior in test/dev ... see permitted_parameters_test.rb
+      rescue_from(StrongerParameters::InvalidParameter) do |e|
+        render json: { error: e.message }, status: :bad_request # uncovered
+      end
+    end
+  end
 end
 
 ActionController::Parameters.include StrongerParameters::Parameters
 ActionController::Base.include StrongerParameters::ControllerSupport
+ActionController::API.include StrongerParameters::APIControllerSupport
```

Comment:
because base is not in api right ?

Reply:
Correct, API is a stripped down version of Base.